apps/asap-server/src/controllers/calendars.ts

```diff
@@ -28,6 +31,30 @@ export default class Calendars implements CalendarController {
     };
   }
 
+  async fetchByResouceId(resourceId: string): Promise<RestCalendar> {
```

Comment:
for this function signature to be valid you need to make `resourceId` unique

Reply:
That's a nice catch, will do! 😉